### ISSUE TITLE
Incorporate information from the weekly email to on-duty CMs into DARIA!

### ISSUE THREAD

**Issue by harumhelmy** [DCAFEngineering/dcaf_case_management#1863]
Hello beloved DARIA team! 

**TL;DR:** I propose we build an Info tab that admins/DCAF CM team can populate with latest information re: clinics, new budget policies, new sibling fund policies, budget policies, etc. 

**The problem**:
Every Sunday evening, DCAF case managers who are getting ready to start their shifts get a really, really long email from the CM leadership team with new information, old reminders, sibling fund and national fund policy updates, DARIA tips, etc. Observe:
![looongemail](https://user-images.githubusercontent.com/13320420/69201700-359a3380-0b0d-11ea-907b-eda5625a536b.gif)
No one wants to read that! Also CMs already have to have many tabs open when they're on shift! What if we incorporate some of this information in the email under an Info tab in DARIA for Users to see, which Admins can populate and publish (through a form that only Admins can access)?

* **How could we solve this issue? (Not knowing is okay!)**
I dunno yet! It's a new feature! 

* **Anything else?**
This is just an idea dump right now, but I'll keep updating it and adding more details. Please feel free to ask clarifications and reach out with any question/comments.

**Comment by colinxfleming** [comment]
first off, holy goodness is that email long.

Here are some more productive reactions. My head goes a few different places on this:

* do other funds issue regular communiques like this, or is this primarily DCAF? how often do the contents of that change? Particularly interested in whether CAF or BRAF do it (BAF lifted their program from DCAF more heavily and aren't a good indicator of whether this is a general practice or not, in my opinion). If it's just DCAF, that's not a reason not to do it, but much of our roadmap these days is oriented toward general utility.
* is this something the CM leadership team thinks would be helpful? Why? 
* we're not feature-complete, but we're kinda close. As we know, things that are in rails become engineering responsibilities to maintain forever, so we need to carefully consider what we put into the app. Even if it sucks to use as an email, if we make it part of DARIA we're strapping some additional weight to our backs, so we want to be extremely deliberate about that.
* the tough thing about this is that it's a LOT of text. my experience has been that people just aren't going to read something of that length no matter how you present it to them, in-app or in-email.
* if we WERE going to roll it out it would be a good excuse to use `ActionText`, the new wysiwyg editor in rails 6!
* would this come off as trying to apply the DCAF process to other funds? we do that a little bit by way of having DARIA at all, but we do try to keep it somewhat process agnostic - would this be forcing a process or look like it? I don't think so, but we do want to carefully consider the politics.

With all that logged, I think the next step here is to do a little bit of user research to find out whether this would be practical and useful. Here are the questions I would ask:

* Do non-other funds send out start-of-shift emails?
* If we had a feature like this would other funds find it useful?
* is this something the DCAF CM directors would find useful, or are they perfectly satisfied with doing this by email?
* if it wasn't email and wasn't DARIA, how would we do this?

I think those questions will give us a sense of whether there's benefit outside of just DCAF, and whether it's worth the additional engineering/maintenance cost.

**Comment by colinxfleming** [comment]
I think this is well overtaken by events at this point
